purchase_order_add_variants/__openerp__.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Author: Hugo Santos
+#    Copyright 2015 FactorLibre
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+{
+    'name': 'Purchase Order Add Variants',
+    'summary': 'Add variants from template into purchase order',
+    'version': '1.0',
+    'author': 'FactorLibre,Odoo Community Association (OCA)',
+    'category': 'Purchase Management',
+    'license': 'AGPL-3',
+    'website': 'http://factorlibre.com',
```

Comment:
https://github.com/OCA/purchase-workflow

Reply:
Follow https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py for the conventions